### ISSUE TITLE
[BITV] Add title to Button component

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/nextcloud-vue#3587]
## Problem

To show a tooltip on a button we currently use v-tooltip. An example can be found in https://github.com/nextcloud/nextcloud-vue/blob/69639353d5994d0decf96dd8d7f53bd559bdb809/src/components/NcAppSidebar/NcAppSidebar.vue#L242-L250. Another example is the `files_sharing` app: https://github.com/nextcloud/server/issues/35654.

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to the Button component that will render as `title` attribute on the HTML element.

**Comment by JuliaKirschenheuter** [comment]
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcButton/NcButton.vue#L201-L202

NcButton will receive `title` attr via other attrs. No changes needed.
